.eslintrc.js

```diff
@@ -9,7 +9,8 @@
     },
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
-        "project": "./tsconfig.json",
+        "project": "./packages/core/tsconfig.json",
```

Comment:
How come the eslintrc file at the root level is referencing a tsconfig file at the package level? Looking at what's done so far, it is the other way around where config files at the package level extend from the root level. Is this intentional?

Reply:
Yes this is intentional since we use our implementation on eslint as of now. Which requires a tsconfig.json. In the follow up task once we migrate to eslint and use nx's lint package, we should be able to get rid of the above.

Reply:
We spoke offline and clarified what I was asking. Came up with a solution for the problem without having to reference package level config at the global level.